GOVERNANCE.md

```diff
@@ -0,0 +1,11 @@
+# Governance
+
+## Project Maintainers
+
+[Project maintainers](CODEOWNERS) are responsible for activities around maintaining and updating CNAB-TO-OCI, a library for the [CNAB spec](https://github.com/cnabio/cnab-spec). Final decisions on the project reside with the project maintainers.
+
+Maintainers MUST remain active. If they are unresponsive for >3 months, they will be automatically removed unless a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of the other project maintainers agrees to extend the period to be greater than 3 months.
```

Comment:
I realize that we probably copied this from elsewhere, but "automatically" implies that there is an automated process which I don't believe we have. 

It may be a good idea to spell out who does the removing. For example, "If they are unresponsive for >3 months, another maintainer may submit a pull request to remove them from CODEOWNERS requiring X approvers unless ..." where X is the usual number of approvers for any other PR.

Otherwise, in my experience people really never get removed because it's awkward and no one knows how to go about it because it's not spelled out.